manifests/common/crd/odhdashboardconfigs.opendatahub.io.crd.yaml

```diff
@@ -178,10 +186,12 @@ spec:
                           type: string
                     storageClassName:
                       type: string
+                # Likely needs a better spot to exit -- hard to do granular permissions
```

Comment:
What do you mean needs a better spot to exit?

Reply:
Tried to address it via [this arch convo](https://redhat-internal.slack.com/archives/C05KDB2HFQQ/p1743442405989389) but the resulting meeting never really got anywhere.

Tried discussing this adhoc -- and less workable. 

Still an unresolved solution... but the original initiative from UX to support this dynamic nature has been indefinitely shelved so it's more of a note than an actionable item at this time.